src/include/planner/operator/ddl/logical_ddl.h

```diff
@@ -5,6 +5,7 @@
 namespace kuzu {
 namespace planner {
 
+// TODO: Now that DDL includes sequence, should rename to not be specified by table
```

Comment:
We should just merge this with LogicalSimple which use to represent operators that only execute once. Do it saperately though